File: src/pucrs/domain/ParticaoMemoria.cs
using System;

namespace Modelos
{
    public class ParticaoMemoria
    {
        public ParticaoMemoria()
        {
            Status = Status.DESALOCADO;
        }

        public Status Status { get; set; }
    }

    public enum Status
    {
        ALOCADO,
        DESALOCADO
    }
}

File: src/pucrs/domain/ParticaoMemoria.java
package pucrs.domain;

public class ParticaoMemoria {
    Status status;

    public ParticaoMemoria() {
        status = Status.DESALOCADO;
    }

    public Status getStatus() {
        return this.status;
    }

    public void setStatus(Status status) {
        this.status = status;
    }
    
    public enum Status {
        ALOCADO, 
        DESALOCADO
    }
}
